### ISSUE TITLE
Ecosystem logos

### ISSUE THREAD

**Issue by bridgetkromhout** [servicemeshinterface/smi-spec.io#20]
These are ecosystem logos for implementations, integrations, etc. Let's get members of the relevant projects to let us know if they want their logos to appear on https://smi-spec.io:
![image](https://user-images.githubusercontent.com/2104453/77119609-ceca4f80-6a04-11ea-8c60-65a86997f7dc.png)

- [x] Consul - @nicholasjackson
- [x] Flagger - @stefanprodan - approved in https://github.com/servicemeshinterface/smi-spec.io/pull/7#issuecomment-597373529
- [ ] Istio
- [x] Linkerd - @grampelberg - approved in https://github.com/servicemeshinterface/smi-spec.io/issues/20#issuecomment-601445406
- [x] Maesh - @dtomcej @SantoDE - approved in https://github.com/servicemeshinterface/smi-spec.io/issues/20#issuecomment-601626362
- [x] Meshery - @leecalcote said to include them in https://github.com/servicemeshinterface/smi-spec.io/pull/7#discussion_r391371231 
- [x] Rio - approved by @smw355 in https://github.com/servicemeshinterface/smi-spec.io/issues/20#issuecomment-601964862
- [x] Service Mesh Hub - @ilevine @bettyjunod - approved in https://github.com/servicemeshinterface/smi-spec.io/issues/20#issuecomment-601457577
~SuperGloo~

**Comment by grampelberg** [comment]
+1 for Linkerd!

**Comment by leecalcote** [comment]
@bridgetkromhout @tomkerkhove the Meshery logo has been updated slightly. Will you kindly use one of the following instead? 
- horizontal ([svg](https://meshery.io/images/logos/meshery-logo-light-text-side.svg)), ([png](https://meshery.io/images/logos/meshery-logo-light-text-side.png)) 
- stacked ([svg](https://meshery.io/images/logos/meshery-logo-light-text.svg)), ([png](https://meshery.io/images/logos/meshery-logo-light-text.png))

**Comment by bettyjunod** [comment]
Update supergloo to Service Mesh Hub logo instead please. Attached are the PNGs. I have high res in .ai or .eps -- can send separately in slack as I cannot upload that filetype into Github 
![meshhub-horizontal](https://user-images.githubusercontent.com/10804749/77122830-da654880-69fb-11ea-9f56-f8864d720171.png)
![meshhub-vertical with solo](https://user-images.githubusercontent.com/10804749/77122835-dcc7a280-69fb-11ea-9c9f-bf2e2e249805.png)

**Comment by tomkerkhove** [comment]
Thank you! #7 is updated.

**Comment by tomkerkhove** [comment]
@bettyjunod I've removed it from the PR but is still listed on https://github.com/servicemeshinterface/smi-spec, should I remove it there as well?

**Comment by bettyjunod** [comment]
@tomkerkhove yes please

**Comment by SantoDE** [comment]
+1 for Maesh.

Logo: 
![maesh](https://user-images.githubusercontent.com/1031255/77154919-f66cf680-6a9c-11ea-9667-56b5c123f3ad.png)

Thank you!

**Comment by smw355** [comment]
+1 for Rio - thanks for asking and including us. 

Shannon Williams

**Comment by bridgetkromhout** [comment]
Thanks, @smw355 - we have Rio's logo on the site now.

**Comment by tomkerkhove** [comment]
Ha you've beat me to it 😀

**Comment by bridgetkromhout** [comment]
@nicholasjackson Can you let us know what logo (if any) you'd like the Consul Connect link on https://smi-spec.io/ to have? Thanks!

**Comment by nicholasjackson** [comment]
Sorry, @bridgetkromhout I swear I pressed the button on this already. But computer says no, I apologise for my tardiness. I have attached the two most recent logos for Consul.

![Consul_VerticalLogo_FullColor](https://user-images.githubusercontent.com/773533/78683536-50681b80-78e7-11ea-9fb4-3847e049c4e2.png)
![Consul_PrimaryLogo_FullColor](https://user-images.githubusercontent.com/773533/78683538-5100b200-78e7-11ea-99d8-51612aa4cfda.png)
